test/fixtures/configfiles/override-config-path.js

```diff
@@ -0,0 +1,5 @@
+var path = require('path');
+
+module.exports = {
+  myappfile: path.join(__dirname, "../override-the-config-path.js"),
```

Comment:
Could we make it possible to specify a relative path from `override-the-config-path.js`? Liftoff should know the absolute path of `override-the-config-path.js`.

Reply:
Do we modify the config returned to users? Since configFiles supports the `extends` property, we'd need to resolve it immediately upon loading otherwise it could be inaccurate

Reply:
Here, I think we can convert the value of `myappfile` from a relative path to an absolute path.
https://github.com/gulpjs/liftoff/pull/128/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346L109

If only absolute paths can be specified, it becomes impossible to specify `myappfile` in the JSON config file included in the project directory, because the project directory can be placed in various directories.

Reply:
Yes, that's where we would add the change. I was wondering if the path normalization would be confusing to users.

I think your suggestion will be good but I'll have to document thoroughly 👍

Reply:
I've made your suggestion, added a test and updated the documentation.

Reply:
I'm going to merge but let me know if you see any issues.